config/kube.json

```diff
@@ -0,0 +1,10 @@
+{
+    "K8sVersion": "1.25.9",
+    "AlternativeImageRepo": "",
+    "ApiserverAdvertiseAddress": "",
+    "PodNetworkCidr": "10.168.0.0/16",
+    "PodNetworkAddonConfigURL": "https://raw.githubusercontent.com/vhive-serverless/vHive/main/configs/calico/canal.yaml",
```

Comment:
is there a reason why we store this config in vHive instead taking from the internet?

Reply:
We are using this file from vhive because vhive's version is alredy very different from the stock one. I agree that this will be the problem for both vhive and invitro to update to newer versions, but it comes from vhive's side currently, we should solve it there first.